### PR TITLE
support of L.esri.DynamicMapLayer in esri-leaflet.

### PR DESCRIPTION
**Pull request by petttro**
Added support of L.esri.DynamicMapLayer in leaflet-image.

### CODE REVIEW

**Comment by tmcw** [comment]
Have you tested this code with just Leaflet? The unguarded reference to `L.esri.dynamicMapLayer` will likely cause a crash, since `L.esri` will be undefined in non-esri environments.

**Comment by petttro** [comment]
Yes, you right, I haven't tested this. It is not working without reference to esri-leaflet.
I'll add validation for this case.

File: leaflet-image.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ module.exports = function leafletImage(map, callback) {
     // layers are drawn in the same order as they are composed in the DOM:
     // tiles, paths, and then markers
     map.eachLayer(drawTileLayer);
+    map.eachLayer(drawEsriDynamicLayer);
```
Comment (tmcw) [inline leaflet-image.js:33]: This code should be included in index.js, not leaflet-image.js: leaflet-image.js is a generated file.

**Review by tmcw — CHANGES_REQUESTED** [review]

File: leaflet-image.js (hunk 1 above)
Comment (petttro) [reply leaflet-image.js:33]: Sorry. 
I have moved changes to index.js and also have added check for L.esri is defined.

Comment (petttro) [reply leaflet-image.js:33]: https://github.com/petttro/leaflet-image/blob/gh-pages/index.js

**Comment by tmcw** [comment]
👍

**Comment by tmcw** [comment]
Published in v0.3.0, thanks @petttro !

**Comment by petttro** [comment]
npm tests were failed.
have fixed issues: https://github.com/petttro/leaflet-image/commit/89cc04f5c4b797ac460768714157857b7fd3b718

54:6   error  Expected indentation of 4 space characters but found 5  indent
 55:9   error  Expected indentation of 9 space characters but found 8  indent
 57:9   error  Expected indentation of 9 space characters but found 8  indent
 60:5   error  Expected indentation of 5 space characters but found 4  indent
 249:10  error  Missing semicolon                                       semi

**Pull request merged**